### PR TITLE
Template symfony rector

### PR DESCRIPTION
**Pull request by mssimi**
#291

### CODE REVIEW

**Comment by mssimi** [comment]
I already discussed it with @TomasVotruba, I will add more commits:)

File: tests/Rector/Contrib/Symfony/HttpKernel/TemplateAnnotationRector/Wrong/wrong8.php.inc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    /**
+     * @Template("AdminBundle:Payment:create.html.twig")
+     */
+    public function indexAction()
+    {
+        return $this->render('payment/new.html.twig', array(
+            'form' => $form->createView(),
+        ));
+    }
+}
```
Comment (TomasVotruba) [inline tests/Rector/Contrib/Symfony/HttpKernel/TemplateAnnotationRector/Wrong/wrong8.php.inc]: Cherry picked to #291, thank you

**Comment by TomasVotruba** [comment]
Merged manually with own code into #291

**Comment by TomasVotruba** [comment]
Thank you

**Pull request closed without merging**